docs/taskbackground.rst

```diff
@@ -1,19 +1,49 @@
+The ``ROOT``-file structure can be represented as a schematic:
+
+.. image:: images/input_structure.png
+  :width: 60%
+  :alt: Diagram showing the structure of each ``ROOT`` file with separate fields for electrons, muons, and jets.
+
+This diagram shows only those fields that will be required for further analysis, such as electron, muon, and jet. 
+Each of these branches has its number of particles (:math:`N_e`, :math:`N_{\mu}`, :math:`N_{jet}`), the transverse momentum value (:math:`p_T`) that will be used in the following sections for events filtering. 
+Also, jets have a b-tag value, which is the output of a discriminator used to identify b-jets (jets produced by b-quark).
+
+The analysis task involves selecting events from the input dataset in which measured quantities originate from :math:`t\bar{t}` decay. 
+In real data, one cannot know with 100\% certainty that an event comes from a specific process, including :math:`t\bar{t}` decay. Through our simulated data, we can ascertain the truth information of each event, which provides us extra tools to develop our analysis. 
+Our data is separated by file into five channels: **ttbar**, **single_top_s_chan**, **single_top_t_chan**, **single_top_tW**, and **wjets**).
+  
+2. Event Selection
 ---------------------------------------------------------------
-Implementations of analysis tasks in the Analysis Grand Challenge aim to demonstrate the integration of different software components in a realistic physics analysis. The task chosen for CMS open data is a measurement of the :math:`t\bar{t}` (top-antitop quark pair production) cross-section  measurement using events with at least four jets and one lepton in the final state. This signature has two possibilities, depending on whether the lepton/neutrino pair originates from the top or antitop quark:
+Not all event information is needed for this analysis. 
+The only objects to which selection criteria will be applied are leptons (electrons and muons) and jets that are the products of :math:`t\bar{t}` decay. 
+In the semi-leptonic decay channel of :math:`t\bar{t}` production, two jets, two b-jets, and one lepton are expected, as can be concluded from the diagram below:
 
 .. image:: images/ttbar.png
   :width: 80%
   :alt: Diagram of a :math:`t\bar{t}` event with four jets and one lepton.
 
-The jets are illustrated in purple. These are collimated showers of hadrons resulting from quarks, which cannot exist on their own due to colour confinement.
+Here is an example of the above signature in CMS Open Data. 
+Since this is Monte Carlo simulated data, we can look at the particles that the jets originate from. 
+The jets are plotted as circles in the :math:`\eta`-:math:`\phi` plane, and color-coordinated with the truth particles they are matched to. Note that :math:`\phi` is a circular variable, so the top of the graph matches to the bottom. 
```

Comment:
Just as an idea here, unrelated to this PR: in principle the phi plot should have an axis range of [-pi, pi] and have ticks along the axis in units of pi as well to really drive home the message what kind of variable this is.

Reply:
I can go ahead and make this edit for this PR if you think it will be beneficial

Reply:
It would affect the notebook too, right? Not crucial here, but feel free to do a follow up PR.

Reply:
That works too.